cmd/mailbox-controller/controller.go

```diff
@@ -269,6 +281,8 @@ func (ctl *mbCtl) ensureEdgeBinding(ctx context.Context, workspace *tenancyv1alp
 		return true
 	}
 	logger = logger.WithValues("mbwsCluster", mbwsCluster, "bindingName", TheEdgeBindingName)
+	//TODO discuss: we can use space-aware client here, but because APIBinding is a third-party API,
+	//              we will need to call ConfigForSpace() and create clientset for each MBWS.
```

Comment:
Yes.  "third-party APIs" will be the norm.  We will eventually need to do this when we switch to using kube-bind.  We could use the space-aware client here right now, or wait 'till later.